Documentation/getting-started.md

```diff
@@ -28,10 +28,11 @@ The Consent for Edge Network extension depends on the following extensions:
 1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file:
 
 ```gradle
-implementation 'com.adobe.marketing.mobile:core:2.+'
-implementation 'com.adobe.marketing.mobile:edge:2.+'
-implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
-implementation 'com.adobe.marketing.mobile:edgeconsent:2.+'
+implementation(platform("com.adobe.marketing.mobile:sdk-bom:3.+"))
+implementation("com.adobe.marketing.mobile:core")
+implementation("com.adobe.marketing.mobile:edge")
+implementation("com.adobe.marketing.mobile:edgeidentity")
+implementation("com.adobe.marketing.mobile:edgeconsent")
```

Comment:
can we remove the parenthesis and use a format similar to our documented examples - https://developer.adobe.com/client-sdks/resources/migration/android/migrate-to-2x/#gradle

Reply:
My vote is to update to use the Kotlin syntax instead of the Groovy syntax.

Reply:
My vote is also on using Kotlin since we converted gradle files to it.